packages/components/src/components/icons/SymphonyAnalytics.tsx

```diff
@@ -6,12 +6,12 @@ const SvgSymphonyAnalytics = (props: SVGProps<SVGSVGElement>) => (
     width="1em"
     height="1em"
     viewBox="0 0 16 16"
-    fill="none"
-    xmlns="http://www.w3.org/2000/svg"
     {...props}
   >
-    <path d="M1.678 7A6.4 6.4 0 0 1 7 1.678V.062A8 8 0 0 0 .062 7zm0 2H.062A8 8 0 0 0 7 15.938v-1.616A6.4 6.4 0 0 1 1.678 9M0 8.04v-.079zM14.322 9h1.616A8 8 0 0 1 9 15.938v-1.616A6.405 6.405 0 0 0 14.322 9m1.616-2h-1.616A6.4 6.4 0 0 0 9 1.678V.062A8 8 0 0 1 15.938 7M8 0h.012z" fill="currentColor"/>
-    <path d="M10 8a2 2 0 0 1-2 2 2 2 0 0 1-2-2 2 2 0 0 1 4 0" fill="currentColor" />
+    <path
+      d="M1.678 7A6.4 6.4 0 0 1 7 1.678V.062A8 8 0 0 0 .062 7zm0 2H.062A8 8 0 0 0 7 15.938v-1.616A6.4 6.4 0 0 1 1.678 9M0 8.04v-.079zM14.322 9h1.616A8 8 0 0 1 9 15.938v-1.616A6.405 6.405 0 0 0 14.322 9m1.616-2h-1.616A6.4 6.4 0 0 0 9 1.678V.062A8 8 0 0 1 15.938 7M8 0h.012zM10 8a2 2 0 0 1-2 2 2 2 0 0 1-2-2 2 2 0 0 1 4 0"
```

Comment:
Why have you updated the icons ? It doesn't seem to be related to React update, does it ?
If no, could you please create another PR for the icon updates ? Thanks

Reply:
The icons updated because I ran `yarn build`. The reason is that the underlying `.svg` icon was updated but the built React Icon has never been committed. We should run a `yarn build` check and perhaps fail if there are file changes.